### PR TITLE
added in documentation for installation in windows

### PR DESCRIPTION
**Pull request by AkshatGadhwal**
addressing issue #1211 
I read the documentation and tried to install bat on windows. in windows, we always have to run 
`choco install bat ` in command prompt opened as administrator. So, this will be good to say this in documentation.

### CODE REVIEW

**Comment by SuperSandro2000** [comment]
Can you remove all the other changes except the one mentioned in the readme?

File: README.md [hunk 1]
```diff
@@ -166,13 +174,13 @@ Also, note that this will [not work](https://github.com/sharkdp/bat/issues/1145)
...
 
-
 ## Installation
 
 [![Packaging status](https://repology.org/badge/vertical-allrepos/bat.svg)](https://repology.org/project/bat/versions)
 
 ### On Ubuntu (using `apt`)
-*... and other Debian-based Linux distributions.*
+
+_... and other Debian-based Linux distributions._
```
Comment (eth-p) [inline README.md:183]: Is there any particular reason that you're replacing the `*` emphasis character here with `_`? If I recall correctly, they mean the same thing in every Markdown specification except Slack/Discord.

File: README.md [hunk 2]
```diff
@@ -304,13 +315,19 @@ You will need to install the [Visual C++ Redistributable](https://support.micros
 #### With Chocolatey
 
 You can install `bat` via [Chocolatey](https://chocolatey.org/packages/Bat):
+
 ```bash
 choco install bat
 ```
 
+if you get an error like
```
Comment (eth-p) [inline README.md:323]: To keep it consistent with the rest of the README, please capitalize "if".

File: README.md [hunk 3]
```diff
@@ -304,13 +315,19 @@ You will need to install the [Visual C++ Redistributable](https://support.micros
...
+"Access to the path 'C:\ProgramData\chocolatey\lib\Less\legal' is denied."
+then run your comman prompt as administration (https://stackoverflow.com/questions/55600258/fix-chocolatey-access-is-denied)
```
Comment (eth-p) [inline README.md:325]: Typos:

"comman prompt"
"as administration" (should be "as administrator"?)

File: README.md [hunk 4]
```diff
@@ -549,7 +572,7 @@ your `PATH` or [define an environment variable](#using-a-different-pager). The [
 ### Colors
 
 Windows 10 natively supports colors in both `conhost.exe` (Command Prompt) and PowerShell since
-[v1511](https://en.wikipedia.org/wiki/Windows_10_version_history#Version_1511_(November_Update)), as
+[v1511](<https://en.wikipedia.org/wiki/Windows_10_version_history#Version_1511_(November_Update)>), as
```
Comment (eth-p) [inline README.md:575]: Although that worked fine before, that's pretty neat. I didn't know Jekyll/GitHub supported escaping URLs with parentheses like that.

File: README.md (hunk 3 above)
Comment (AkshatGadhwal) [reply README.md:325]: sorry for typos, i will correct them

**Comment by sharkdp** [comment]
Thank you very much for your contribution.

This is not really something that is specific to `bat`, right? Chocolatey even mentions the "Access to path" error in their troubleshooting section: https://chocolatey.org/docs/troubleshooting#access-to-the-path-is-denied

To be honest, I'd rather not add package-manager specific content to our README, especially if it's not related the `bat` package at all.

**Pull request closed without merging**